src/Microsoft.ComponentDetection.Detectors/ruby/RubyComponentDetector.cs

```diff
@@ -142,8 +144,8 @@ private void ParseGemLockFile(ISingleFileComponentRecorder singleFileComponentRe
             else
             {
                 // Throw this line away. Is this malformed? We were expecting a header
-                this.Logger.LogVerbose(lines[0]);
-                this.Logger.LogVerbose("Appears to be malformed/is not expected here. Expected heading.");
+                this.Logger.LogDebug("{MalformedLine}", lines[0]);
+                this.Logger.LogDebug("Appears to be malformed/is not expected here.  Expected heading. {Line}", lines[0]);
```

Comment:
There is an extra space added here
```suggestion
                this.Logger.LogDebug("Appears to be malformed/is not expected here. Expected heading. {Line}", lines[0]);
```